### PR TITLE
 Internal: Docs - Container [ED-7047]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: docs/modules/container-converter/container-converter.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+This is a migration tool for Elementor that helps users migrating from the older Sections/Columns layouts to the new and
+updated Container layouts.
+
+It's a standalone module that injects conversion buttons to each Section & Page, and will (or at least should) be
+removed in the future when the Container element will be a stable feature and become the default building block.
+
+It registers 2 commands:
+
+- `$e.run( 'container-converter/convert' )` - Converts a single "convertable" element and its descendants to a Containers.
```
Comment (ManorHazaz) [inline docs/modules/container-converter/container-converter.md:17]:
```suggestion
- `$e.run( 'container-converter/convert', { container } )` - Converts a single "convertable" element and its descendants to a Containers.
```

**Review by Nevoss — APPROVED** [review]

File: docs/includes/elements/container.md [hunk 2]
```diff
@@ -74,3 +76,149 @@ Another aspect is the new elements hierarchy. We needed to decide which elements
...
+```
+
+This variables reset method is required in order to avoid style leakage as stated above.
+
+#### Guidelines:
+
+- Name the CSS variable after the corresponding CSS property.
+- Try to always use `{{WRAPPER}}` as the selector, and consume the variable under the proper selector in the SCSS file
+  instead of putting a long selector in the PHP file. 
+- Make sure the default value you set for the property is the real default according to
```
Comment (matipojo) [inline docs/includes/elements/container.md:129]: If the element has a default value (for example, align=left) it should be used for the CSS variable initial value.

File: docs/modules/container-converter/container-converter.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+## How It Works:
+
+The PHP side of things is fairly easy. It just handles injecting the conversion buttons and loading the assets.
+
+On the frontend from the other hand (pun intended), things get more complicated.
+
+We can generally divide that into 2 parts:
+
+1. **Migrator** - Handles the actual migration logic (duh...) using a simple API.
```
Comment (matipojo) [inline docs/modules/container-converter/container-converter.md:30]:
```suggestion
1. **Migrator** - Handles the actual migration logic  using a simple API.
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/includes/elements/container.md (hunk 2 above)
Comment (StyleShit) [reply docs/includes/elements/container.md:129]: Yup, done.

File: docs/modules/container-converter/container-converter.md (hunk 3 above)
Comment (StyleShit) [reply docs/modules/container-converter/container-converter.md:30]: 🤣 
Done

**Pull request merged**